src/components/sortingFunctions.js

```diff
@@ -20,6 +20,30 @@ export const getShoppingItemBackgroundStyles = (
   }
 };
 
+//#047857 #10B981 #6EE7B7
+
+export const getShoppingItemTextStyles = (
+  daysLeftForNextPurchase,
+  getDaysBetweenCurrentAndPreviousPurchase,
+  lastPurchasedOn,
+) => {
+  const presentDate = Date.now();
+  const daysBetweenCurrentAndPreviousPurchase = getDaysBetweenCurrentAndPreviousPurchase(
+    lastPurchasedOn,
+    presentDate,
+  );
+
+  if (daysBetweenCurrentAndPreviousPurchase >= daysLeftForNextPurchase * 2) {
+    return '#111827';
+  } else if (daysLeftForNextPurchase <= 7) {
+    return '#F3F4F6';
+  } else if (daysLeftForNextPurchase < 30) {
+    return '#111827';
+  } else {
+    return '#111827';
+  }
```

Comment:
Instead of hardcoding the colors here you could use [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) and store the values in your main css file. 
Or instead of returning colors you could return class names, and store the styles there.

Reply:
Awesome! Now if you ever want to change the color of the dark text, you won't have to worry if you forgot to update a value somewhere :smiley: